### PR TITLE
Test/Full Attribute deletion process

### PR DESCRIPTION
**Pull request by Milena-Czierlinski**
These are the tests I added in #185 that actually are independent of default Attribute. I didn't bump the version, since I only add tests here.

### CODE REVIEW

File: packages/consumption/test/modules/attributes/AttributesController.test.ts [hunk 1]
```diff
@@ -591,6 +591,126 @@ describe("AttributesController", function () {
                 expect(deletedChildAttribute).toBeUndefined();
             }
         });
+
+        describe("should validate and execute full attribute deletion process", function () {
+            let predecessorRepositoryAttribute: LocalAttribute;
+            let successorRepositoryAttribute: LocalAttribute;
+            let predecessorOwnSharedIdentityAttribute: LocalAttribute;
+            beforeEach(async () => {
```
Comment (jkoenig134) [inline packages/consumption/test/modules/attributes/AttributesController.test.ts:599]:
```suggestion
            let predecessorOwnSharedIdentityAttribute: LocalAttribute;

            beforeEach(async () => {
```

**Review by britsta — COMMENTED** [review]
I assume you have already incorporated the comments that were given to these tests in the PR https://github.com/nmshd/runtime/pull/185 here? 👻

**Comment by Milena-Czierlinski** [comment]
> I assume you have already incorporated the comments that were given to these tests in the PR #185 here? 👻

Of course!

**Review by jkoenig134 — DISMISSED** [review]

File: packages/consumption/test/modules/attributes/AttributesController.test.ts [hunk 2]
```diff
@@ -591,6 +591,126 @@ describe("AttributesController", function () {
...
+                    content: IdentityAttribute.from({
+                        value: {
+                            "@type": "EMailAddress",
+                            value: "my-new@email.address"
+                        },
+                        owner: consumptionController.accountController.identity.address
+                    }),
+                    shareInfo: {
+                        peer: CoreAddress.from("peer"),
+                        requestReference: CoreId.from("reqRef2"),
```
[the remark this answers is not in the document]
Comment (Milena-Czierlinski) [reply packages/consumption/test/modules/attributes/AttributesController.test.ts:640]: Indeed

**Review by britsta — APPROVED** [review]

**Review by jkoenig134 — APPROVED** [review]

**Pull request merged**